canonical-web-and-design/canonicalwebteam.blog
# Improve related posts

## Done
Increase the compatibility of related articles by increasing the sample set to 30 articles and returning the top three matching articles with the most matching tags.

## QA
- Check out this branch: https://github.com/canonical-web-and-design/ubuntu.com/pull/9947
- Run the site locally (demos don't work)
- Join the VPN
- Go to https://localhost:8001/blog
- Check a few blogs and see the related articles have more matching tags

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9934
